ci-operator/config/openshift/console-plugin-template/openshift-console-plugin-template-main.yaml

```diff
@@ -0,0 +1,56 @@
+base_images:
+  nodejs-16:
+    name: nodejs-16
+    namespace: openshift
+    tag: latest
+  nginx:
```

Comment:
@jhadvig Added `ngnix` to base images since we are mirroring it too, and it is plugin-template dependency.

Reply:
this looks correct 👍